crates/web-plugins/oob-messages/src/models.rs

```diff
@@ -112,7 +118,7 @@ where
 
     let did = diddoc.id.clone();
     let oob_message = OobMessage::new(&did);
-    let url: &String = &format!("{}:{}", server_public_domain, server_local_port);
+    let url: &String = &format!("{}", server_public_domain);
     let oob_url = OobMessage::serialize_oob_message(&oob_message, url)
```

Comment:
The conversion here is unnecessary.
```suggestion
    let oob_url = OobMessage::serialize_oob_message(&oob_message, server_public_domain)
```